packages/cozy-harvest-lib/src/datacards/useDataCardFiles.js

```diff
@@ -21,46 +21,52 @@ const useFolderToSaveFiles = folderToSaveId =>
     }
   )
 
-const useSourceAccountFiles = accountId =>
+const useSourceAccountIdentifierFiles = sourceAccountIdentifier =>
   useQuery(
     Q('io.cozy.files')
-      .where({ 'cozyMetadata.sourceAccount': accountId, trashed: false })
-      .indexFields(['cozyMetadata.sourceAccount', 'cozyMetadata.createdAt'])
+      .where({
+        'cozyMetadata.sourceAccountIdentifier': sourceAccountIdentifier,
+        trashed: false
+      })
+      .indexFields([
+        'cozyMetadata.sourceAccountIdentifier',
+        'cozyMetadata.createdAt'
+      ])
       .sortBy([
-        { 'cozyMetadata.sourceAccount': 'desc' },
+        { 'cozyMetadata.sourceAccountIdentifier': 'desc' },
         { 'cozyMetadata.createdAt': 'desc' }
       ])
       .limitBy(5),
     {
-      as: `fileDataCard_io.cozy.accounts/${accountId}/io.cozy.files`,
+      as: `fileDataCard_io.cozy.accounts.sourceAccountIdentifier/${sourceAccountIdentifier}/io.cozy.files`,
```

Comment:
Voir : https://github.com/cozy/cozy-guidelines?tab=readme-ov-file#naming-of-queries
```
io.cozy.files/sourceAccountIdentifier/${sourceAccountIdentifier}
```

Reply:
Ok